if: update page (#7481)
tldr-pages/tldr: pages/common/if.md

# if

> Performs conditional processing in shell scripts.
> See also: `test`, `[`.
> More information: <https://www.gnu.org/software/bash/manual/bash.html#Conditional-Constructs>.

> - Execute the specified commands if the condition command exit status is zero:

Reviewer: ```suggestion
- Execute the specified commands if the condition command's exit status is zero:
```

`if {{condition_command}}; then {{commands}}; fi`

> - Execute the specified commands if the condition command exit status is nonzero:

Reviewer: ```suggestion
- Execute the specified commands if the condition command's exit status is nonzero:
```

> - Execute the specified commands if the condition command's exit status is nonzero:

Reviewer 2: Perhaps this will read better?
```suggestion
- Execute the specified commands if the condition command's exit status is not zero:
```

`if ! {{condition_command}}; then {{commands}}; fi`

> - Execute the first specified commands if the condition command exit status is zero otherwise execute the second specified commands:

Reviewer: ```suggestion
- Execute the first specified commands if the condition command's exit status is zero otherwise execute the second specified commands:
```

> - Execute the first specified commands if the condition command's exit status is zero otherwise execute the second specified commands:

Reviewer 2: Perhaps this will read better?
```suggestion
- Execute the first set of commands if the condition command's exit status is zero, otherwise execute the second set of commands:
```

Author: It seems that code snippet description is going to become very verbose. ;) Personally I like your suggestion but I have to think about when it's better to use smth like "list/set of" in command descriptions to write more standardized tldr pages. If anybody has any thoughts please tell me them, I am glad to hear them. ;)
